docs/Installing.asciidoc

```diff
@@ -246,68 +265,49 @@ installed 'os-autoinst' from packages make sure to pass +--isotovideo+ option
 to point to the checkout dir where isotovideo is, not to +/usr/lib+! Otherwise
 it will have trouble finding its perl modules.
 
-[[authentication]]
-=== User authentication
-
-OpenQA supports three different authentication methods - OpenID (default), iChain
-and Fake. See +auth+ section in +/etc/openqa/openqa.ini+.
-
-[source,ini]
---------------------------------------------------------------------------------
-[auth]
-# method name is case sensitive!
-method = OpenID|iChain|Fake
---------------------------------------------------------------------------------
+== Where to now?
 
-Independently of method used, the first user that logs in (if there is no admin yet)
-will automatically get administrator rights!
+From this point on, you can refer to the <<GettingStarted.asciidoc#get-testing,Getting Started>> guide to
+fetch the tests cases and possibly take a look at <<WritingTests.asciidoc#writingtests,Test Developer Guide>>
 
-=== OpenID
+== Advanced configuration
+[id="advanced"]
 
-By default openQA uses OpenID with opensuse.org as OpenID provider.
-OpenID method has its own +openid+ section in +/etc/openqa/openqa.ini+:
 
-[source,ini]
---------------------------------------------------------------------------------
-[openid]
-## base url for openid provider
-provider = https://www.opensuse.org/openid/user/
-## enforce redirect back to https
-httpsonly = 1
---------------------------------------------------------------------------------
-
-OpenQA supports only OpenID version up to 2.0. Newer OpenID-Connect and OAuth is
-not supported currently.
+[[database]]
```

Comment:
Database configuration is not part of advanced configuration, please move it back :) (and keep it before Authentication)

Reply:
I don't understand why it is basic configuration.

I consider basic configuration something required for the _thing_ to work. All the other steps are required to configure even with default values, the database not.

Reply:
@binary-sequence If you don't set up the database, you won't be able to run openQA unless you install the openqa-local-db package, which is very specific to openSUSE.